src/common/adapters/jira.js

```diff
@@ -13,43 +13,40 @@ import match from 'micro-match';
 
 import client from '../client';
 
-const DOMAIN = '.atlassian.net';
-
-const issueTabMatch = '/projects/:project/issues/:id';
-const browseIssueMatch = '/browse/:id';
-
-function selectedIssue({ href, pathname }) {
-  let id = null;
+function isJiraPage(loc, doc) {
+  if (loc.host.endsWith('.atlassian.net')) return true;
+  if (doc.body.id === 'jira') return true;
+  return false;
+}
```

Comment:
👍 Maybe we can update the documentation of the adapter then

Reply:
Not sure what you mean? This just checks whether we're on a Jira page or not. The issue key/id retrieval still works the same as before.

Reply:
Nevermind, I meant the maybe we can specify that the adapter can work also with a self-hosted instance. But I think is not really needed.